src/sdl2/messagebox.rs

```diff
@@ -48,3 +101,60 @@ pub fn show_simple_message_box(flags: MessageBoxFlag, title: &str,
         Err(SdlError(get_error()))
     }
 }
+
+/// returns the id of the button triggered
+pub fn show_message_box<'a>(flags:MessageBoxFlag, buttons:&'a [ButtonData], title:&str,
```

Comment:
Is a button id really the best thing to return from this? My heart tells me what we really want is a reference to one of the borrowed buttons we feed in, but I've never really used message boxes so I'm not all that sure.